src/api/ml/z3native_stubs.c.pre

```diff
@@ -294,7 +294,8 @@ static struct custom_operations Z3_ast_plus_custom_ops = {
   Z3_ast_compare_ext
 };
 
-MK_CTX_OF(ast, 16) // let's say 16 bytes per ast
+// FUDGE
+MK_CTX_OF(ast, 8) // let's say 16 bytes per ast
```

Comment:
"let's say?" 8 is the new 16?

Reply:
ah, need to remove the comment :). These are size estimations, that the GC uses to guestimate how heavy a foreign value is. Seems like giving OCaml smaller estimations makes it faster here, it might be that it started garbage collecting like crazy with the previous change.